bedrock/contentful/api.py

```diff
@@ -591,20 +580,15 @@ def get_info_data(self, entry_obj, seo_obj=None):
         return data
 
     def get_content(self):
-        # Check if it is a page or a connector, or a Compose page type
-
+        # Check if it is a page or a connector
         entry_type = self.page.content_type.id
         seo_obj = None
-        if entry_type == COMPOSE_MAIN_PAGE_TYPE:
-            # Contentful Compose page, linking to content and SEO models
-            entry_obj = self.page.content  # The page with the actual content
-            seo_obj = self.page.seo  # The SEO model
-            # Note that the slug lives on self.page, not the seo_obj.
-        elif entry_type.startswith("page"):
-            entry_obj = self.page
-        elif entry_type == "connectHomepage":
+        if entry_type == "connectHomepage":
             # Legacy - TODO: remove me once we're no longer using Connect: Homepage
             entry_obj = self.page.fields()["entry"]
+        elif entry_type.startswith("page"):  # WARNING: this reuires a consistent naming of page types in Contentful, too
```

Comment:
```suggestion
        elif entry_type.startswith("page"):  # WARNING: this requires a consistent naming of page types in Contentful, too
```

I think this is a reasonable naming convention to enforce